app/scripts/modules/core/src/presentation/forms/layouts/BasicLayout.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { IFieldLayoutProps } from '../interface';
+
+export class BasicLayout extends React.Component<IFieldLayoutProps> {
```

Comment:
An example `Layout` component.  A Layout takes individual react elements and renders them in the right places.  For example, a different layout might use flexbox (or bootstrap or tables) for layout, or it might move the validation section to the right instead of the bottom.

Reply:
We can definitely use the basic layout I added from Semantic for this purpose. It handles inputs, text areas, help areas, combined fields, and a lot more. What do you think? 
https://github.com/spinnaker/styleguide/blob/master/src/styles/forms.css#L12 
Adapted from https://semantic-ui.com/collections/form.html